<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Asistente</class>
 <widget class="QWidget" name="Asistente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>131</width>
     <height>261</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../resource.qrc">:/images/asistente.png</pixmap>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>0</y>
     <width>20</width>
     <height>391</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 127);</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>7</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>251</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Posicion de los dedos: (x,y,z,t)</string>
   </property>
   <widget class="QLineEdit" name="posicionDedosXLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="posicionDedosTLineEdit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="posicionDedosZLineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="posicionDedosYLineEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>261</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Temperatura de los dedos: (x,y,z,t)</string>
   </property>
   <widget class="QLineEdit" name="temperaturaDedosXLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="temperaturaDedosTLineEdit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="temperaturaDedosYLineEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="temperaturaDedosZLineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="vacioLineaSuperiorLineEdit">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>190</y>
     <width>151</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="vacioLineaInferiorLineEdit">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>230</y>
     <width>151</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="vacioCamaralabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>150</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Vacio de la Camara:</string>
   </property>
  </widget>
  <widget class="QLabel" name="vacioLineaInflabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>230</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Vacio de la Linea Inferior:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="vacioCamaralineEdit">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>150</y>
     <width>151</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="vacioLineaSuplabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>190</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Vacio de la Linea Superior:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="temperaturaNeveraLineEdit">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>270</y>
     <width>151</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="tempNeveralabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>270</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Temperatura de la Nevera:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="humedadDomoLineEdit">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>354</y>
     <width>151</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="humedadDomoLabel_4">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>354</y>
     <width>171</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Humedad del domo:</string>
   </property>
  </widget>
  <widget class="QLabel" name="gradoslabel_2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>320</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>(Grados celcius)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="temperaturaDomoLineEdit">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>318</y>
     <width>151</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="temperaturaDomoLabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>318</y>
     <width>171</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Temperatura del domo:</string>
   </property>
  </widget>
  <widget class="QLabel" name="porclabel">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>360</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="gradoslabel_3">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>275</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>(Grados celcius)</string>
   </property>
  </widget>
  <widget class="QLabel" name="gradoslabel_4">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>235</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>(Grados celcius)</string>
   </property>
  </widget>
  <widget class="QLabel" name="gradoslabel_5">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>190</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>(Grados celcius)</string>
   </property>
  </widget>
  <widget class="QLabel" name="gradoslabel_6">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>155</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>(Grados celcius)</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
